pkgs/games/tetrio-desktop/default.nix

```diff
@@ -0,0 +1,82 @@
+{ stdenv
+, lib
+, fetchurl
+, autoPatchelfHook
+, alsa-lib
+, cups
+, libX11
+, libXScrnSaver
+, libXtst
+, mesa
+, nss
+, systemd
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tetrio-desktop";
+  version = "8.0.0";
+
+  src = fetchurl {
+    url = "https://tetr.io/about/desktop/builds/TETR.IO%20Setup.deb";
+    name = "${pname}.deb";
+    sha256 = "1nlblfhrph4cw8rpic9icrs78mzrxyskl7ggyy2i8bk9i07i21xf";
+  };
+
+  nativeBuildInputs = [
+    alsa-lib
+    autoPatchelfHook
+    cups
+    libX11
+    libXScrnSaver
+    libXtst
+    mesa
+    nss
+    wrapGAppsHook
+  ];
+
+  dontWrapGApps = true;
+
+  libPath = lib.makeLibraryPath [
+    alsa-lib
+    cups
+    libX11
+    libXScrnSaver
+    libXtst
+    mesa
+    nss
+    systemd
+  ];
+
+  unpackPhase = ''
+    mkdir -p $TMP/${pname} $out/bin
+    cp $src $TMP/${pname}.deb
+    ar vx $TMP/${pname}.deb
```

Comment:
substituting pname a lot makes the code harder to understand.

Reply:
would you like me to substitute all of them with `tetrio-desktop`?

Reply:
I personally find that easier to understand.

Reply:
alright, changed